snuba/utils/streams/backends/kafka.py

```diff
@@ -615,26 +617,100 @@ def closed(self) -> bool:
 
 DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = 50000
 DEFAULT_QUEUED_MIN_MESSAGES = 10000
+DEFAULT_PARTITIONER = "consistent"
+DEFAULT_MAX_MESSAGE_BYTES = 50000000  # 50MB, default is 1MB
+SUPPORTED_KAFKA_CONFIGURATION = (
+    # Check https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+    # for the full list of available options
+    "bootstrap.servers",
+    "sasl.mechanism",
+    "sasl.username",
+    "sasl.password",
+    "security.protocol",
+)
+DEFAULT_STORAGE_NAME = ""
+
+
+def get_default_kafka_configuration(
+    storage_name: Optional[str] = None,
+    bootstrap_servers: Optional[Sequence[str]] = None,
+    override_params: Optional[Mapping[str, Any]] = None,
+) -> KafkaBrokerConfig:
+    if storage_name in settings.DEFAULT_STORAGE_BROKERS:
```

Comment:
Potentially someone could add a None key in the DEFAULT_STORAGE_BROKERS and that would be a valid key. So please don't look into DEFAULT_STORAGE_BROKERS and STORAGE_BROKER_CONFIG if the storage name you receive here is None. So we have a defined way of loading the default config where no storage is available.

Also please refer to storages with the `StorageKey` type instead of the bare string so mypy can type check this. 
```
storage_name: Optional[StorageKey] = None,
```

Reply:
will do